src/index.js

```diff
@@ -0,0 +1,78 @@
+import _ from 'lodash';
+import './style.css';
+import TaskObject from '../modules/taskObj';
+import AddDelete from '../modules/tasks';
+import Store from '../modules/store';
+import StatusUpdate from '../modules/statusUpdate';
+
```

Comment:
I suggest you to not leave any unnecessarily blank lines in your code. This makes your code unclean and complex. Kindly fix them and remove the unnecessarily blank lines from your code.